### PR TITLE
Clean and upgrade go dependencies

### PR DESCRIPTION
**Pull request by hectorj2f**
I upgraded our go dependencies which were creating some errors when building PRs via Travis.

Error: https://travis-ci.org/giantswarm/mayu/builds/158505697

ping @giantswarm/team-positive

### CODE REVIEW

File: vendor/github.com/BurntSushi/toml/.travis.yml [hunk 1]
```diff
@@ -2,11 +2,14 @@ language: go
 go:
   - 1.1
   - 1.2
+  - 1.3
+  - 1.4
+  - 1.5
+  - 1.6
```
Comment (xh3b4sd) [inline vendor/github.com/BurntSushi/toml/.travis.yml]: What happens when you provide multiple go versions here? Is there a build run with every version each? I would say we only run using one version, or at least the 3 most recent go versions.

Comment (hectorj2f) [inline vendor/github.com/BurntSushi/toml/.travis.yml]: Are u telling me that it executes the travis.yml of all the go dependencies :D ?

Comment (xh3b4sd) [inline vendor/github.com/BurntSushi/toml/.travis.yml]: I don't really understand why the go dependency property is a list. I would interpret this as "build this project for every version listed here". In fact There are only single builds in the travis web interface. I would only use the latest go version and that is it.

Comment (xh3b4sd) [inline vendor/github.com/BurntSushi/toml/.travis.yml]: Ignore me. I realized that his is not our `.travis.yml`. All good.

**Comment by xh3b4sd** [comment]
The changes of this PR LGTM. I cannot validate if mayu still works as expected though.

**Comment by teemow** [comment]
LGTM and WFM

**Comment by hectorj2f** [comment]
Thanks @teemow

**Pull request merged**